docs/ref/django-admin.txt

```diff
@@ -601,12 +601,13 @@ The :djadmin:`dumpdata` command can be used to generate input for ``loaddata``.
 Compressed fixtures
 ~~~~~~~~~~~~~~~~~~~
 
-Fixtures may be compressed in ``zip``, ``gz``, or ``bz2`` format. For example::
+Fixtures may be compressed in ``zip``, ``gz``, ``bz2`` or ``xz`` format.
```

Comment:
Serial comma should stay before 'or' (see https://en.wikipedia.org/wiki/Serial_comma).